app/config/factionhall/events.json

```diff
@@ -1,60 +1,52 @@
 {
     "events": [
         {
-            "name": "The Politics of Moral Panic: How Polarization Undermined Collective Reason by Lucien Greaves",
-            "description": "Join Lucien Greaves to discuss The Politics of Moral Panic: How Polarization Undermines Collective Reason.",
+            "name": "Sober Faction Memorial Ritual",
+            "description": "Co-Director Priest Joe Dee & Reverend Jon Eldritch with Sober Faction's Council will kick off the event with a memorial ritual for all those lost to overdose.",
             "room": "auditorium",
-            "start": "2022-08-20T20:00:00-04:00",
-            "end": "2022-08-20T21:00:00-04:00",
-            "image": "https://tstvhq.s3.amazonaws.com/backgrounds/factionhall/greyfaction/calendarhall.png"
+            "start": "2022-08-31T20:00:00-04:00",
+            "end": "2022-08-31T21:00:00-04:00",
+            "image": "https://tstvhq.s3.amazonaws.com/backgrounds/factionhall/Sober+Faction+2022/SF+CALENDAR.jpeg"
         },
         {
-            "name": "The Rise and Fall of Bennett Braun by Evan Anderson",
-            "description": "Join Grey Faction director Evan Anderson to discuss The Rise and Fall of Bennett Braun.",
-            "room": "rotunda",
-            "start": "2022-08-20T21:00:00-04:00",
-            "end": "2022-08-20T22:00:00-04:00",
-            "image": "https://tstvhq.s3.amazonaws.com/backgrounds/factionhall/greyfaction/calendarhall.png"
+            "name": "Naloxone Training",
+            "description": "Minister Kal Set & Director of Sober Faction Outreach, Keleigh Rose, will host a Naloxone Training.",
+            "room": "jude-mirra",
+            "start": "2022-08-31T21:00:00-04:00",
+            "end": "2022-08-31T22:00:00-04:00",
+            "image": "https://tstvhq.s3.amazonaws.com/backgrounds/factionhall/Sober+Faction+2022/SF+CALENDAR.jpeg"
         },
         {
-            "name": "The Limitations in the Treatment of Satan Within In-Depth Psychology by Prof. Amanda Barton",
-            "description": "Join Professor Amanda Barton to discuss The Limitations in the Treatment of Satan Within In-Depth Psychology.",
-            "room": "hall",
-            "start": "2022-08-20T22:00:00-04:00",
-            "end": "2022-08-20T23:00:00-04:00",
-            "image": "https://tstvhq.s3.amazonaws.com/backgrounds/factionhall/greyfaction/calendarhall.png"
+            "name": "Open Share Sober Faction Meeting",
+            "description": "An open share Sober Faction meeting will take place for all to remember loved ones.",
+            "room": "large-lounge",
+            "start": "2022-08-31T22:00:00-04:00",
+            "end": "2022-08-31T23:00:00-04:00",
+            "image": "https://tstvhq.s3.amazonaws.com/backgrounds/factionhall/Sober+Faction+2022/SF+CALENDAR.jpeg"
```

Comment:
If there are issues with this, I'd blame the spaces left in the filenames that are replaced with plus signs. If you end up getting stuck later, that's where I'd fix